examples/enforcement/breach.json

```diff
@@ -0,0 +1,397 @@
+      "responses": [
+        {
+          "value": "No, none of the above"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Is the property in Medway?",
+      "responses": [
+        {
+          "value": "Yes"
+        }
+      ],
+      "metadata": {
+        "autoAnswered": true
+      }
+    },
+    {
+      "question": "Has the property or site already been granted planning permission for the works you're reporting?",
+      "responses": [
+        {
+          "value": "Yes, it's been granted planning permission but I still think it's a breach"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Enter your contact details",
+      "responses": [
+        {
+          "value": "Ebenezer Scrooge 0123456789 scrooge@waltdisney.com"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Enter your address",
+      "responses": [
+        {
+          "value": "16 Bayham Street, Camden Town, London, NW1 0JU, UK"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Is the breach currently in progress?",
+      "responses": [
+        {
+          "value": "Yes"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Select the breach you are reporting",
+      "responses": [
+        {
+          "value": "New or modified outbuilding (e.g. garage or shed)"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Describe the work suspected to be a breach",
+      "responses": [
+        {
+          "value": "Erection of a library in the rear garden"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Do you know when the work started?",
+      "responses": [
+        {
+          "value": "Yes"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "When did work start?",
+      "responses": [
+        {
+          "value": "2024-12-25"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Do you know when the work was completed?",
+      "responses": [
+        {
+          "value": "No"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Which side of the property is the outbuilding on?",
+      "responses": [
+        {
+          "value": "Front"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Do you know any of the dimensions of the outbuilding?",
+      "responses": [
+        {
+          "value": "The library is approximately 4x3 metres"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "How many other outbuildings are there?",
+      "responses": [
+        {
+          "value": "1"
+        }
+      ],
+      "metadata": {}
+    },
+    {
+      "question": "Do you have any supporting documents or photos?",
+      "responses": [
+        {
+          "value": "No"
+        }
+      ],
+      "metadata": {}
```

Comment:
It's not required to upload any files - hences `files?` in the main type definition. If files are uploaded, they'll be labeled according to our usual types